djongo/base.py

```diff
@@ -85,20 +85,21 @@ def is_usable(self):
 
     def get_connection_params(self):
         # TODO: Add other parameters here
-        if not self.settings_dict['NAME']:
-            return 'djongo_test'
-            # from django.core.exceptions import ImproperlyConfigured
-            # raise ImproperlyConfigured(
-            #     "settings.DATABASES is improperly configured. "
-            #     "Please supply the NAME value.")
-        return self.settings_dict['NAME']
-
-    def get_new_connection(self, db_name):
+        settings_dict = {}
+
+        settings_dict['name'] = self.settings_dict.get('NAME') or 'djongo_test'

```

Comment:
Can u change this line and others to:

```
settings_dict['name'] = self.settings_dict.get('NAME', 'djongo_test')
```

Reply:
Nice, I wasn't aware of that usage of the get method. I'll do it.